assets/js/timetree.js

```diff
@@ -8,119 +8,136 @@ import {schemeGreens} from "d3-scale-chromatic"
 const d3 = {select, forceSimulation, forceManyBody, forceCenter, forceCollide,
     forceLink, forceY, line, curveNatural, scaleSequential, schemeGreens};
 
+// load & parse leaf data from json embedded in the document
+const leafData = document.querySelector('.leaf-data');
+const data = JSON.parse(leafData.value);
+
+// generate list of centuries referenced in the data; sort most recent first
+let centuries = Array.from(new Set(
+  data.leaves.filter(leaf => leaf.century != undefined)
+      .map(leaf => leaf.century))
+  ).sort().reverse();
```

Comment:
does precommit format this? just curious. kind of slick

Reply:
ooh thanks for reminder, I haven't setup precommit formatting yet but I really should